contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -159,6 +159,19 @@
           this.initImageFields();
         }
       },
```

Comment:
Those two new watchers are related to a situation when a new file is dropped to the editor area (when uploading an image using the image upload menu, progress and errors will be displayed - I haven't checked yet but can see that implementation is there)

Reply:
Follow-up issue https://github.com/learningequality/studio/issues/2177